.travis.yml

```diff
@@ -43,22 +43,22 @@ env:
   - secure: ULUg4jjil5NND94FARuMbXS3cpS6pCIcu793tKTRaNvXAnpo5XHGd/VNStmWZez5EswA8NqI6KEWjl/0nnROWTkQXHczsJjebkzfhRwsqc2De+u3HZ+qAO1DieYQP++XGvvzWyry4ejFfy3qVC11GCV6CIJqMizQUt/D3QHHa3bwL8Tw8PubYsCW2wQa2lwKSc8Kw3Tj3x/pL/8Nw74dJ9M+Cyeh4XZq0Pp6Wlxkxe/zRPKydhV/DihT2rLEiMpxWsXMX1OS1cnXUJUvPf/bdwXMQMocZc3rjs/CMWWn7bx/k5Wdue481xGOdulIu9ljR/ZrxY+5xxm5a3UZd+ZYB8Y5tghiln4k7XDtdUXTSTBzZoDfRoboUbIiPzI0gma8XQJTeLXViS5T6p4M/SdUHkf5s0w8zJKRamqGDI0TL6wI3BnL85VQwXhMjWEmC5Zi1EZHgAGStFDxRGG2TLIHzFuSYly5zuTifyl80bcSj9czgkd6WoPKT3NKPsNTbWpQNF2Amixg4L5xr3Re5zFkTcpyaa05kySE9w84vqxPkczwKIDCmBuoTz6q1+rWaajM4KqlweNKo7OwPGy1R2zvZSq9vSa2a+f5YYpvUvjIYhvUpMQs1B0chC5kAFA2aTMXi+HMMabXq8AShoq8bnO7tBbPUxiu8OST72UQrF0j9Bg=
 matrix:
   include:
-  - python: '3.4'
+  - python: '3.6'
     env: NPM=Test
-  - python: '3.4'
+  - python: '3.6'
     env: UNIT=Test
 notifications:
   slack:
     secure: DQyUTk6evwPpO0P5+OPhBSgl+fGEerOjBlBwQliXAkDaMKH3Cpi4cTQ3ETR/+3g6bGqPGK+QJ1R+6Ht+hJD7dNomyVIoQmvF0P5afJtpk/A3cDILe90t76ET9jc/iBjWeUFQdokFUJ7Gt1GGYtI6e5XcVu/Qc/xqCvFMtsBN6mnBFuvcQki+WkoIIPPawyhfryCNOLo5vvbi4SZ5dZI+M0MGq9HQjOyF1sdIgKuXhxjupmL+kVPVXAq9kiOANYFkwbNsP9j0BTYW5wFHpAztBqz3NT6EchgCdue8tgV4hC4rRFvM/bsA4qz/TJ5wRRLBRXtnNtPcyhAqTiU+wC6qWt++fjSEMGe4KYqtRRV1YuxQiTVHN84t77DILLNfMh/6uSs0KijwOT+Oazwd/UsN1zYOH93AM4FZ0h92yyb6j6JQ+DUk4WFel1Zr4kZzhtHSPGw+K4fxY0zIt1qpaDPXjtZHQ0+LwIIMtwMp5bBcwDn9d1ADnUhUAuuIN2hHaXrVy4vP2hIcd0LzezBqvc7JXimyd5yRgUeOCTrKGkAeSo8VA7XIj0ZmlpQRYKNTJP+gz4Y6C/RCxISnFDF/vcX+IsDdvreZXJMplE1Aqxf0uR6Zj8Sr8q+QWGKydv6ettlLZuqDuv0l/l/9qpzYfRqLSZcetGRVFHHcR9wuQiDyums=
 before_install:
 - python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-- SNO_PATH="snovault = git https://github.com/$USER/$SNO_REPO.git rev="
-- SNO_BRANCH=$(grep "${SNO_PATH}" buildout.cfg | sed "s@$SNO_PATH@@")
-- SNO_STATUS=$(curl -s "https://api.travis-ci.org/$USER/$SNO_REPO.svg?branch=$SNO_BRANCH"
-  | grep pass)
-- ! "if [ -z \"$SNO_STATUS\" ]; then\n    echo \"Snovault branch build for $SNO_BRANCH
-  is failing; exiting build\"\n    travis_terminate\nelse\n    echo \"Snovault branch
-  $SNO_BRANCH is okay with build status: $SNO_STATUS\"\nfi\n"
+# - SNO_PATH="snovault = git https://github.com/$USER/$SNO_REPO.git rev="
+# - SNO_BRANCH=$(grep "${SNO_PATH}" buildout.cfg | sed "s@$SNO_PATH@@")
+# - SNO_STATUS=$(curl -s "https://api.travis-ci.org/$USER/$SNO_REPO.svg?branch=$SNO_BRANCH"
+#   | grep pass)
+# - ! "if [ -z \"$SNO_STATUS\" ]; then\n    echo \"Snovault branch build for $SNO_BRANCH
+#   is failing; exiting build\"\n    travis_terminate\nelse\n    echo \"Snovault branch
+#   $SNO_BRANCH is okay with build status: $SNO_STATUS\"\nfi\n"
```

Comment:
Uncomment and delete. We do not care if snovault build passes, it just slows us down.